### DISCUSSION TITLE
Support for bluesky?

### DISCUSSION THREAD

**Discussion by steven-tan** [s9e/TextFormatter#234] [Ideas]
I don't suppose you have plans to add bluesky support?

https://bsky.app/

I know it's not exactly replacing twitter yet, but one can dream lol!

**Comment by JoshyPHP** [comment]
I plan to add it in the hopefully-not-too-distant future but there's a couple of things regarding the implementation that I haven't decided yet.

**Reply by steven-tan** [reply]
Thank you!
